client/src/components/Overview/ProductDetail.jsx

```diff
@@ -1,30 +1,45 @@
 import { StyledProductDetail } from '../styles/Overview/ProductDetail.styled';
 const ProductDetail = ({description, slogan, features}) => {
+
+  const shareToSocialMedia = (socialMedia) => {
+    alert(`Shared to ${socialMedia}`);
```

Comment:
Is this supposed to do anything else with the social media?

Reply:
Doc says the following,
"buttons should provide the ability to share this product on popular social media platforms"
There are some libraries that can help to do this, I am not sure if we can use them or not. If not, I am not sure if we want to implement one ourselves.

Reply:
Hm yeah, might not be worth figuring out right now